halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/deployment/v1/DistributedDeployer.java

```diff
@@ -400,6 +405,10 @@ public RemoteAction deploy(DistributedServiceProvider<T> serviceProvider,
   private <T extends Account> Set<Integer> reapOrcaServerGroups(AccountDeploymentDetails<T> details,
       SpinnakerRuntimeSettings runtimeSettings,
       DistributedService<Orca, T> orcaService) {
+    if (runtimeSettings.getServiceSettings(orcaService.getService()).getSkipLiveCycleManagement()) {
+      return Collections.EMPTY_SET;
```

Comment:
`return new Collections.emptySet()` is I think the preferred way

Reply:
Looks to be return Collections.emptySet();

https://docs.oracle.com/javase/8/docs/api/java/util/Collections.html#emptySet--